### PR TITLE
Add Project Management

### PR DESCRIPTION
**Pull request by NathanJesudason**
Just adding in the start of the project management section (what is in the current PM repo). Includes a basic explanation of the task tracker.

### CODE REVIEW

**Comment by calebeby** [comment]
Deploy preview ready!

Built with commit 7a083a831be2ce773138a938c0273ff8869be3be

https://deploy-preview-19--pigmice.netlify.com

**Comment by NathanJesudason** [comment]
Interesting... the deploy doesn't seem to recognize the changes I made. Any idea what the problem is?

**Comment by calebeby** [comment]
Works better if you spell `management` right :wink: 

```diff
- [Project Management](project-managment)
+ [Project Management](project-management)
```

**Comment by NathanJesudason** [comment]
Thanks, just fixed that. I am also noticing that the markdown syntax isn't translating. Am I missing something obvious or are all of these files somehow not .md files? (It is just header stuff mostly, I could just remove it)

**Comment by calebeby** [comment]
I think you need to put a space after `#`

**Review by howardabrams — DISMISSED** [review]

**Comment by calebeby** [comment]
@ErgoShrimp Is this ready? If so, please add the `ready` label. Thanks!

File: project-management/trello.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Trello
```
Comment (calebeby) [inline project-management/trello.md:1]: Could we add a screenshot here?

**Review by calebeby — CHANGES_REQUESTED** [review]

**Review by calebeby — APPROVED** [review]
Thanks!

**Pull request merged**